syntax = "proto3";

package sample;

message HelloRequest {
  string name = 1;
}

message HelloReply {
  string message = 1;
}

service HelloService {
  rpc SayHello (HelloRequest) returns (HelloReply) {}
}